### PR TITLE
feat: add content-list

### PR DESCRIPTION
**Pull request by yuberdysheva**
content component size L and S: https://preview.gravity-ui.com/page-constructor/468/?path=/story/components-content--size

Content Layout Block: https://preview.gravity-ui.com/page-constructor/468/?path=/story/blocks-contentlayout--default

Media Block: https://preview.gravity-ui.com/page-constructor/468/?path=/story/blocks-media--default

### CODE REVIEW

File: src/sub-blocks/Content/ContentList/ContentList.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    return (
+        <div className={b({size})}>
+            {contentList?.map((item) => {
+                const {icon, title, text} = item;
+                const iconThemed = getThemedValue(icon, theme);
+                const iconData = getMediaImage(iconThemed);
+
+                return (
+                    <div className={b('item')} key={title || text}>
```
Comment (NikitaCG) [inline src/sub-blocks/Content/ContentList/ContentList.tsx:30]: if we dont have title or text we dont have key, and  we'll have error with this example 
`contentList = [{icon: '...'}, {icon: '...'}], {icon: '...'}]]`

File: src/sub-blocks/Content/Content.tsx [hunk 2]
```diff
@@ -72,6 +75,7 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
                     />
                 </div>
             )}
+            {contentList && <ContentList contentList={contentList} size={size} />}
```
Comment (NikitaCG) [inline src/sub-blocks/Content/Content.tsx:78]: maybe just check the length of the array and not even render the component if it's empty?

**Review by NikitaCG — CHANGES_REQUESTED** [review]

File: src/sub-blocks/Content/ContentList/ContentList.tsx (hunk 1 above)
Comment (NikitaCG) [reply src/sub-blocks/Content/ContentList/ContentList.tsx:30]: you can use `import {v4 as uuidv4} from 'uuid';` for key

**Review by NikitaCG — DISMISSED** [review]

File: src/sub-blocks/Content/Content.tsx [hunk 3]
```diff
@@ -72,6 +75,7 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
                     />
                 </div>
             )}
+            {contentList?.length && <ContentList contentList={contentList} size={size} />}
```
Comment (gorgeousvlad) [inline src/sub-blocks/Content/Content.tsx:78]: if contentList is empty array,  `0` will be rendered here

File: src/components/MediaBase/MediaBaseContent.tsx [hunk 4]
```diff
@@ -21,6 +30,7 @@ const MediaContent = (props: MediaContentProps) => {
                 links={links}
                 theme="default"
                 buttons={allButtons}
+                contentList={contentList}
```
Comment (gorgeousvlad) [inline src/components/MediaBase/MediaBaseContent.tsx:33]: `contentList` -> `items`

File: src/sub-blocks/Content/ContentList/ContentList.tsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+import {ThemeValueContext} from '../../../context/theme/ThemeValueContext';
+import {ContentItemProps, ContentSize} from '../../../models';
+import {block, getThemedValue} from '../../../utils';
+
+import './ContentList.scss';
+
+const b = block('content-list');
+
+interface ContentListProps {
+    contentList: ContentItemProps[];
```
Comment (gorgeousvlad) [inline src/sub-blocks/Content/ContentList/ContentList.tsx:15]: contentList -> items

**Review by gorgeousvlad — CHANGES_REQUESTED** [review]

File: src/sub-blocks/Content/Content.tsx [hunk 6]
```diff
@@ -75,7 +75,7 @@ const Content = (props: ContentBlockProps & ClassNameProps) => {
                     />
                 </div>
             )}
-            {list?.length && <ContentList list={list} size={size} />}
+            {list?.length && Boolean(list.length) && <ContentList list={list} size={size} />}
```
Comment (gorgeousvlad) [inline src/sub-blocks/Content/Content.tsx:78]: `list?.length ? <ContentList list={list} size={size} /> : null`

**Review by gorgeousvlad — APPROVED** [review]

**Pull request merged**